test/port.js

```diff
@@ -65,8 +65,70 @@ describe('port', () => {
 
 		it('should return a  promise', () => {
 			var p = port.connect();
-			expect(p).to.be.defined;
-			expect(p.then).to.be.defined;
+			expect(p).to.not.be.undefined;
+			expect(p.then).to.not.be.undefined;
```

Comment:
The latter doesn't exist and wasn't doing anything. (This test should have been failing)